apps/src/code-studio/components/progress/UnitOverview.jsx

```diff
@@ -75,9 +76,11 @@ class UnitOverview extends React.Component {
       props.redirectScriptUrl && props.redirectScriptUrl.length > 0;
     this.state = {showRedirectDialog};
 
-    analyticsReporter.sendEvent(EVENTS.UNIT_OVERVIEW_PAGE_VISITED_EVENT, {
-      'unit name': props.scriptName
-    });
+    if (props.userType === 'teacher') {
+      analyticsReporter.sendEvent(EVENTS.UNIT_OVERVIEW_PAGE_VISITED_EVENT, {
```

Comment:
I would suggest also updating the event name to reflect that it is only for teachers. this will have the side benefit of making it clear which data is good (teacher only) and which data is bad (included students)

Reply:
This suggest sounds great. Turner can you make the change in the code and on amplitude? "Unit Overview Page Visited By Teacher"

Reply:
Yes will do!